functions/Remove-DbaDbData.ps1

```diff
@@ -0,0 +1,166 @@
+function Remove-DbaDbData {
+    <#
+    .SYNOPSIS
+        Removes all the date from a database(s) for each instance(s) of SQL Server.
```

Comment:
date > data

Reply:
I read through this so many times 😆  .. thank you!